[2022-08-16 15:20:09,720] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: yandex-rates-test.insert_data manual__2022-08-16T15:19:54.120072+00:00 [queued]>
[2022-08-16 15:20:09,753] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: yandex-rates-test.insert_data manual__2022-08-16T15:19:54.120072+00:00 [queued]>
[2022-08-16 15:20:09,754] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-16 15:20:09,755] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-08-16 15:20:09,756] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-16 15:20:09,803] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): insert_data> on 2022-08-16 15:19:54.120072+00:00
[2022-08-16 15:20:09,817] {standard_task_runner.py:52} INFO - Started process 6740 to run task
[2022-08-16 15:20:09,839] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'yandex-rates-test', 'insert_data', 'manual__2022-08-16T15:19:54.120072+00:00', '--job-id', '34', '--raw', '--subdir', 'DAGS_FOLDER/yandex_dag.py', '--cfg-path', '/tmp/tmp5xzbv_04', '--error-file', '/tmp/tmpkxmquiyd']
[2022-08-16 15:20:09,891] {standard_task_runner.py:80} INFO - Job 34: Subtask insert_data
[2022-08-16 15:20:10,104] {task_command.py:371} INFO - Running <TaskInstance: yandex-rates-test.insert_data manual__2022-08-16T15:19:54.120072+00:00 [running]> on host b232b3e4166c
[2022-08-16 15:20:10,256] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yandex-rates-test
AIRFLOW_CTX_TASK_ID=insert_data
AIRFLOW_CTX_EXECUTION_DATE=2022-08-16T15:19:54.120072+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-16T15:19:54.120072+00:00
[2022-08-16 15:20:10,279] {logging_mixin.py:115} INFO - rate_date:  2022-08-16
[2022-08-16 15:20:10,281] {logging_mixin.py:115} INFO - value_:  24207.92857100000037
[2022-08-16 15:20:10,297] {base.py:68} INFO - Using connection ID 'postgre_test_db' for task execution.
[2022-08-16 15:20:10,337] {python.py:173} INFO - Done. Returned value was: None
[2022-08-16 15:20:10,369] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=yandex-rates-test, task_id=insert_data, execution_date=20220816T151954, start_date=20220816T152009, end_date=20220816T152010
[2022-08-16 15:20:10,451] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-16 15:20:10,507] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
